### PR TITLE
Add feedback

### PR DESCRIPTION
**Pull request by jenwei**
### Changes
- Add minor copy edits
- Reformat racer-basics
- Add links to README

### Feedback
- Would be nice for the tutorial descriptions to live closer to the individual tutorials, possibly in their own READMEs, especially if you plan on adding more details to them
- Do we want to keep the pronouns consistent? Notice occasional `I`s though `we` is used the majority of the time
- Is there a reason why the comments/instructions within the HTML are comments? Feels like something OK to have show in browser so people don't have to toggle between the HTML and browser as much. 
    - For someone new to Derby, I'd expect the flow to be something like this 👇 
        - Go through setup
        - Go to some page (e.g. /hello-world)
        - Try it out
        - Go into source code to understand how it works
    -  . . . but it's more like this 👇
        - Go through setup
        - Go to some page (e.g. /hello-world)
        - Maybe type things
        - Go to README to figure out what I'm supposed to do
        - Go to source code to see what to do
        - Do stuff
        - Go into source code to understand how it works
- What's `wholesome-component`? Is that supposed to map to `Components in action`?
- Is there supposed to be a tutorial of some sort for `Collections and data persistence`?

### CODE REVIEW

**Comment by jenwei** [comment]
Can't seem to assign this to you, so @-mentioning you here for review, @hdavidzhu 🤦‍♀️

File: README.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ Take a look at `src/index.js` for all the different routes that are available. W
 
 ## Contributing
 
-If you find anything confusing, please [submit an issue](https://github.com/lever/derby-tutorial/issues), or [submit a PR](https://github.com/lever/derby-tutorial/pulls) with tutorial improvements! Your help and feedback is really appreciated :heart:
+If you find anything confusing, please [submit an issue here](https://github.com/hdavidzhu/derby-tutorial/issues), or submit a PR with tutorial improvements! Your help and feedback is really appreciated :heart:
```
Comment (hdavidzhu) [inline README.md]: woop, looks like there was a small revert here

File: src/components/racer-basics/index.html [hunk 2]
```diff
@@ -1,40 +1,36 @@
 <index:>
-  <div clas="row">
-    <div class="col">
-      <pre>
-        {{JSON.stringify(exampleData, null, 2)}}
-      </pre>
-    </div>
-    <div class="col">
+  <!-- Learn more about these basic methods @ https://derbyjs.com/docs/derby-0.10/models/setters -->
+  <div class="row">
```
Comment (hdavidzhu) [inline src/components/racer-basics/index.html:3]: 👍

**Comment by hdavidzhu** [comment]
Thanks @jenwei! This is really useful feedback. I totally agree with your sentiment about the tutorial living closer to the source. The tutorial docs displayed in HTML would also be really helpful.

**Pull request merged**